packages/ws-server/tests/acceptance/sendRawTransaction.spec.ts

```diff
@@ -43,22 +44,36 @@ describe('@release @web-socket eth_sendRawTransaction', async function () {
     [WsTestConstant.FAKE_TX_HASH, 'rpc', 'invalid'],
   ];
 
+  // @ts-ignore
+  const { mirrorNode, relay } = global;
+
   let tx: any,
     accounts: AliasAccount[] = [],
     ethersWsProvider: WebSocketProvider;
-
+  let requestId: string;
   before(async () => {
-    accounts[0] = await global.servicesNode.createAliasAccount(100, global.relay.provider);
-    accounts[1] = await global.servicesNode.createAliasAccount(100, global.relay.provider);
-    accounts[2] = await global.servicesNode.createAliasAccount(5, global.relay.provider);
-    await new Promise((r) => setTimeout(r, 1000)); // wait for accounts to propagate
+    requestId = Utils.generateRequestId();
+    const initialAccount: AliasAccount = global.initialAccount;
+    const initialAmount: string = '10000000000'; //100 Hbar
+
+    const neededAccounts: number = 3;
+    accounts.push(
+      ...(await Utils.createMultipleAliasAccounts(
+        mirrorNode,
+        initialAccount,
+        neededAccounts,
+        initialAmount,
+        requestId,
+      )),
+    );
+    global.accounts = accounts;
```

Comment:
We are overriding this in multiple places, isn't this going to be hard to debug, depending on the order of the tests?

Reply:
I've chabned the logic a bit and now we push into the array, instead of overriding.